charts/stable/common/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: common
 description: Function library for k8s-at-home charts
 type: library
-version: 4.3.0
+version: 4.3.1
```

```suggestion
version: 4.3.0
```
This can stay on 4.3.0 because it's the development branch

---

Ok, I removed the version bump.